ff/serving/provider/offline.go

```diff
@@ -125,7 +127,7 @@ type ResourceRecord struct {
 
 func (rec ResourceRecord) check() error {
 	if rec.Entity == "" {
-		return errors.New("ResourceRecord must have Entity set.")
+		return errors.New("resourceRecord must have Entity set.")
```

Comment:
This can stay uppercase since it's the name of an object. You're IDE will probably continue to give you warnings

Reply:
fixed